### ISSUE TITLE
Arches 5.0 can no longer be installed due to broken dependency

### ISSUE THREAD

**Issue by AJ-HistoricEngland** [archesproject/arches#6615]
**Describe the bug**

Arches v5.0 no long runs due to celery 4.3.0 specifying the requirement of `vine>=1.3.0`. The vine package upgraded from 1.3.0 to 5.0.0 on 6th Sept 2020. celery 4.3.0 no longer works.

**To Reproduce**
1. Setup workspace and virtualenv
2. Run `pip install arches`. You will see the the following error: `amqp 2.6.1 requires vine<5.0.0a1,>=1.1.3, but you'll have vine 5.0.0 which is incompatible.`
3. Try to create a new project `python ENV\Scripts\arches-project create test_v500`
```cmd
C:\Development\repos\arches-500
(env) λ python ENV\Scripts\arches-project create test_v500
No module named 'vine.five'
Traceback (most recent call last):
  File "ENV\Scripts\arches-project", line 150, in <module>
    main()
  File "ENV\Scripts\arches-project", line 143, in main
    COMMANDS[args.command](args)
  File "ENV\Scripts\arches-project", line 56, in command_create_app
    cmd.handle(options)
  File "ENV\Scripts\arches-project", line 33, in handle
    self.validate_name(project_name, "project")
  File "C:\Development\repos\arches-500\env\lib\site-packages\django\core\management\templates.py", line 219, in validate_name
    app=app_or_project,
django.core.management.base.CommandError: 'test_v500' is not a valid project name. Please make sure the name is a valid identifier.
```

**Workaround**
A temporary workaround is to reinstall the correct version of vine:

```cmd
 pip uninstall vine
 pip install vine==1.3.0
```

**Expected behavior**

**Your Arches Information**

*   Version used: 5.0
*   Operating System and version (desktop or mobile): Windows 10
*   Browser Name and version: Version 85.0.4183.83 (Official Build) (64-bit)
*   Link to your Arches Install (optional):

**Additional context**

**Ticket Background**
*   Found by: @AJ-HistoricEngland

**Comment by chrisdolby-HE** [comment]
I think this is based upon combined releases to upgrade to Celery 5, amqp 5, and vine 5, based upon what @AJ-HistoricEngland has noted: https://github.com/celery/celery/issues/6266#issuecomment-687967584

**Comment by mradamcox** [comment]
I recently experienced this also when I created a fresh installation of Arches 5 and was unable to run the command to create an app. I can't test this now, but added vine==1.3.0 to the end of requirements.txt will hopefully solve the issue.

**Comment by thedrow** [comment]
Please upgrade to Celery 4.3.1.
This will fix your issue.

**Comment by thedrow** [comment]
Also, since you only support Python 3, feel free to upgrade Celery 5 when it is released. :smile:

**Comment by chiatt** [comment]
Resolved in #6661
